Sources/SwifterSwift/Foundation/NSUbiquitousKeyValueStoreExtensions.swift

```diff
@@ -0,0 +1,18 @@
+// NSUbiquitousKeyValueStoreExtensions.swift - Copyright 2020 SwifterSwift
+
+#if canImport(Foundation) && !os(Linux) && !os(watchOS)
+import Foundation
+
+// MARK: - Methods
+
+public extension NSUbiquitousKeyValueStore {
+    /// SwifterSwift: URL from NSUbiquitousKeyValueStore.
+    ///
+    /// - Parameter forKey: key to find URL for.
+    /// - Returns: URL object for key (if exists).
+    func url(forKey key: String) -> URL? {
+        return URL(string: object(forKey: key) as? String)
```

Comment:
```suggestion
        return URL(string: string(forKey: key))
```

Reply:
Also, probably is good to go with something like:
```swift
func url(forKey key: String) -> URL? {
        guard let strURL = string(forKey: key) else { return nil }
        return URL(string: strURL)
}
```